api/db/migrations/20230928084925_flag_certifications_before_july_21_OUTDATED.js

```diff
@@ -0,0 +1,24 @@
+const TABLE_NAME = 'certification-courses';
+const COLUMN_NAME = 'cpfImportStatus';
+
```

Comment:
Utiliser OUTDATED depuis l'import plutôt qu'une string classique
```suggestion
import { cpfImportStatus }  from '...certification-course.js';
cpfImportStatus.OUTDATED
```